sormas-backend/src/main/java/de/symeda/sormas/backend/facility/FacilityFacadeEjb.java

```diff
@@ -614,4 +637,56 @@ private Facility fillOrBuildEntity(@NotNull FacilityDto source, Facility target,
 
 		return target;
 	}
+
+	private Facility mergeOrBuildEntity(@NotNull FacilityDto source, Facility target, boolean checkChangeDate) {
```

Comment:
Please have a look at DtoHelper.copyDtoValues - that should do what you're doing here, but in a generic way (because right now, this method would need to be adjusted every time we add a new property to facilities).

Sorry for not pointing out the existence of that method in the issue description... didn't think about it myself at that point in time.

Reply:
Done